zappa/wsgi.py

```diff
@@ -197,3 +197,129 @@ def get_wsgi_string(string, encoding="utf-8"):
     Returns wsgi-compatible string
     """
     return string.encode(encoding).decode("iso-8859-1")
+
+
+def create_wsgi_request_from_v2_event(
```

Comment:
Looks like a lot of this code is repeated.

What here needs to be changed that's different from the existing `create_wsgi_request` function?

Reply:
yeah, most of the code is repetitive. but there's some that are different such as getting request context (path, method, body), or getting cookie and authorizer info. I'll try to implement some small functions to extract those duplicate codes

Reply:
@monkut I've merged the implementation to the existing function bcs only few fields that are different